{"data": [
{"status": "RUNNING", "name": "spawnFlop", "run_id": "df64a60463", "start_time": "16-11-22 23:37:46", "project": "treehouse", "donor_id": "10", "pipeline_name": "RNAseq", "sample_id": "10", "error_text": null},
{"status": "FAILED", "name": "spawnFlop", "run_id": "b54c274a8d", "start_time": "16-11-22 23:37:31", "project": "treehouse", "donor_id": "7", "pipeline_name": "RNAseq", "sample_id": "7", "error_text": "\"Runtime error:\\nTraceback (most recent call last):\\n  File \\\"/home/hancock/analysis_core/env/local/lib/python2.7/site-packages/luigi/worker.py\\\", line 192, in run\\n    new_deps = self._run_get_new_deps()\\n  File \\\"/home/hancock/analysis_core/env/local/lib/python2.7/site-packages/luigi/worker.py\\\", line 130, in _run_get_new_deps\\n    task_gen = self.task.run()\\n  File \\\"/home/hancock/analysis_core/dashboard-service/luigi-interface/spawned.py\\\", line 19, in run\\n    raise Exception(\\\"This was a failure.\\\")\\nException: This was a failure.\\n\""},
{"status": "FAILED", "name": "spawnFlop", "run_id": "c9453e7f29", "start_time": "16-11-22 23:37:21", "project": "treehouse", "donor_id": "5", "pipeline_name": "RNAseq", "sample_id": "5", "error_text": "\"Runtime error:\\nTraceback (most recent call last):\\n  File \\\"/home/hancock/analysis_core/env/local/lib/python2.7/site-packages/luigi/worker.py\\\", line 192, in run\\n    new_deps = self._run_get_new_deps()\\n  File \\\"/home/hancock/analysis_core/env/local/lib/python2.7/site-packages/luigi/worker.py\\\", line 130, in _run_get_new_deps\\n    task_gen = self.task.run()\\n  File \\\"/home/hancock/analysis_core/dashboard-service/luigi-interface/spawned.py\\\", line 19, in run\\n    raise Exception(\\\"This was a failure.\\\")\\nException: This was a failure.\\n\""},
{"status": "FAILED", "name": "spawnFlop", "run_id": "ec3594f08d", "start_time": "16-11-22 23:37:00", "project": "treehouse", "donor_id": "1", "pipeline_name": "RNAseq", "sample_id": "1", "error_text": "\"Runtime error:\\nTraceback (most recent call last):\\n  File \\\"/home/hancock/analysis_core/env/local/lib/python2.7/site-packages/luigi/worker.py\\\", line 192, in run\\n    new_deps = self._run_get_new_deps()\\n  File \\\"/home/hancock/analysis_core/env/local/lib/python2.7/site-packages/luigi/worker.py\\\", line 130, in _run_get_new_deps\\n    task_gen = self.task.run()\\n  File \\\"/home/hancock/analysis_core/dashboard-service/luigi-interface/spawned.py\\\", line 19, in run\\n    raise Exception(\\\"This was a failure.\\\")\\nException: This was a failure.\\n\""},
{"status": "FAILED", "name": "spawnFlop", "run_id": "c6068f4861", "start_time": "16-11-22 23:37:10", "project": "treehouse", "donor_id": "3", "pipeline_name": "RNAseq", "sample_id": "3", "error_text": "\"Runtime error:\\nTraceback (most recent call last):\\n  File \\\"/home/hancock/analysis_core/env/local/lib/python2.7/site-packages/luigi/worker.py\\\", line 192, in run\\n    new_deps = self._run_get_new_deps()\\n  File \\\"/home/hancock/analysis_core/env/local/lib/python2.7/site-packages/luigi/worker.py\\\", line 130, in _run_get_new_deps\\n    task_gen = self.task.run()\\n  File \\\"/home/hancock/analysis_core/dashboard-service/luigi-interface/spawned.py\\\", line 19, in run\\n    raise Exception(\\\"This was a failure.\\\")\\nException: This was a failure.\\n\""},
{"status": "FAILED", "name": "spawnFlop", "run_id": "84d6699aee", "start_time": "16-11-22 23:37:41", "project": "treehouse", "donor_id": "9", "pipeline_name": "RNAseq", "sample_id": "9", "error_text": "\"Runtime error:\\nTraceback (most recent call last):\\n  File \\\"/home/hancock/analysis_core/env/local/lib/python2.7/site-packages/luigi/worker.py\\\", line 192, in run\\n    new_deps = self._run_get_new_deps()\\n  File \\\"/home/hancock/analysis_core/env/local/lib/python2.7/site-packages/luigi/worker.py\\\", line 130, in _run_get_new_deps\\n    task_gen = self.task.run()\\n  File \\\"/home/hancock/analysis_core/dashboard-service/luigi-interface/spawned.py\\\", line 19, in run\\n    raise Exception(\\\"This was a failure.\\\")\\nException: This was a failure.\\n\""},
{"status": "DONE", "name": "spawnFlop", "run_id": "21d099ea75", "start_time": "16-11-22 23:37:26", "project": "treehouse", "donor_id": "6", "pipeline_name": "RNAseq", "sample_id": "6", "error_text": null},
{"status": "DONE", "name": "spawnFlop", "run_id": "cbe43d0c6e", "start_time": "16-11-22 23:37:36", "project": "treehouse", "donor_id": "8", "pipeline_name": "RNAseq", "sample_id": "8", "error_text": null},
{"status": "DONE", "name": "spawnFlop", "run_id": "8feeb700d9", "start_time": "16-11-22 23:37:05", "project": "treehouse", "donor_id": "2", "pipeline_name": "RNAseq", "sample_id": "2", "error_text": null},
{"status": "DONE", "name": "spawnFlop", "run_id": "1a4caceeee", "start_time": "16-11-22 23:37:15", "project": "treehouse", "donor_id": "4", "pipeline_name": "RNAseq", "sample_id": "4", "error_text": null},
]}